src/test/java/ezschedule/testutil/EventBuilder.java

```diff
@@ -12,8 +12,8 @@ public class EventBuilder {
 
     public static final String DEFAULT_NAME = "CS2103 Event";
     public static final String DEFAULT_DATE = "2023-05-01";
-    public static final String DEFAULT_START_TIME = "12:00";
-    public static final String DEFAULT_END_TIME = "22:00";
+    public static final String DEFAULT_START_TIME = "08:00";
+    public static final String DEFAULT_END_TIME = "10:00";
```

Comment:
How does the change contribute to the code? Can it remain as it is?

Reply:
Yes it can remain as it is. I initially changed it to due to some testing, changing the default start/end time would not affect the program in any ways.